xcm/pallet-xcm-benchmarks/src/lib.rs

```diff
@@ -56,15 +56,15 @@ pub type ExecutorOf<T> = xcm_executor::XcmExecutor<<T as Config>::XcmConfig>;
 /// The overarching call type.
 pub type OverArchingCallOf<T> = <T as frame_system::Config>::Call;
 /// The asset transactor of our executor
-pub type AssetTransactorOf<T> = <<T as Config>::XcmConfig as xcm_executor::Config>::AssetTransactor;
+pub type AssetTransactorOf<T> = <<T as Config>::XcmConfig as XcmConfig>::AssetTransactor;
 /// The call type of executor's config. Should eventually resolve to the same overarching call type.
-pub type XcmCallOf<T> = <<T as Config>::XcmConfig as xcm_executor::Config>::Call;
+pub type XcmCallOf<T> = <<T as Config>::XcmConfig as XcmConfig>::Call;
 
-pub fn mock_worst_case_holding() -> MultiAssets {
-	const HOLDING_FUNGIBLES: u32 = 99;
-	const HOLDING_NON_FUNGIBLES: u32 = 99;
+pub fn mock_worst_case_holding(depositable_count: u32, max_assets: u32) -> MultiAssets {
```

Comment:
While merging master, this is one of the things that I've changed in order to get the code to compile, and I'm not certain whether this line (and this file in general, especially the part where I added `depositable_count` to `Config::worst_case_holding`) is the correct fix.

May want to review this a little closer.